perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java

```diff
@@ -1149,7 +1172,7 @@ public void getUsersByIds() throws Exception {
 			users.add(user2);
 		}
 
-		assertEquals(users, perun.getUsersManagerBl().getUsersByIds(sess, ids));
+		assertEquals(users, perun.getUsersManager().getUsersByIds(sess, ids));
```

Comment:
When you are at it, can you please fix this test? When we use real DB, there is no guarantee, that returned list has same order as checked one, resulting in false positive test failure.
You can gather users on input to set and you can do the same for the result of getUsersByIds(). Then you can call assertEquals() on two sets, where order doesn't matter. Thank you.

Reply:
Done

Reply:
Thank you.